kubectl-plugin/test/e2e/kubectl_ray_test.go

```diff
@@ -0,0 +1,39 @@
+package e2e
+
+import (
+	"os/exec"
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+func TestKubectlRayGet(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Kubectl Ray Get")
+}
+
+var _ = Describe("Calling ray plugin `get` command", Ordered, func() {
+	It("succeed in getting ray cluster information", func() {
+		cmd := exec.Command("kubectl", "ray", "cluster", "get")
+		output, err := cmd.CombinedOutput()
+
+		Expect(err).NotTo(HaveOccurred())
+		Expect(output).To(ContainSubstring("raycluster-sample"))
+	})
+
+	It("should not succeed", func() {
+		cmd := exec.Command("kubectl", "ray", "cluster", "get", "fakeclustername", "anotherfakeclustername")
+		output, err := cmd.CombinedOutput()
+
+		Expect(err).To(HaveOccurred())
+		Expect(output).ToNot(ContainElements("fakeclustername"))
+	})
+
+	It("will have a `namespace` and should succeed  without error", func() {
```

Comment:
This check and the first one is pretty similar. Can we just remove this one and add a namespace flag to the first one?